client/data/settings/resolvers.js

```diff
@@ -22,9 +22,39 @@ export function* getSettings() {
 	try {
 		const result = yield apiFetch( { path } );
 		yield updateSettings( result );
+
+		const surveyResult = yield apiFetch( {
```

Comment:
I just put it here because it was easier to submit on the settings page for testing, so that I could also avoid auth.
This is not going to be submitted on mount of the settings page.